calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -38,7 +38,32 @@ The geeky details of what you get:
   details='Policy:Calico,IPAM:AWS,CNI:AWS,Overlay:No,Routing:VPC Native,Datastore:Kubernetes'
 />
 
-To enable {{prodname}} network policy enforcement on an EKS cluster using the AWS VPC CNI plugin, follow these step-by-step instructions: [Installing Calico on Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/calico.html)
+First, create an Amazon EKS cluster.
+```bash
+eksctl create cluster --name  my-calico-cluster
```

Comment:
```suggestion
eksctl create cluster --name  <my-calico-cluster>
```
Syntax for user-supplied values.

UPDATE: I see that you're just following the format from elsewhere in the doc. Feel free to leave this. Alternatively, for extra credit, change globally.

Reply:
Also: To format the code block as part of the step, make sure that it is aligned with the first character of the step. So the first backtick of the code fence is in the same column as the 'F' in 'First'. Format similarly below.